shared/chat/conversation/messages/message-popup/exploding/container.js

```diff
@@ -122,6 +129,7 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     if (stateProps._canEdit) {
       items.push({onClick: dispatchProps._onEdit, title: 'Edit'})
     }
+    items.push({onClick: dispatchProps._onCopy(message), title: 'Copy text'})
```

Comment:
this sets `onClick` to the return value of `dispatchProps._onCopy`. you need a closure like `() => dispatchProps._onCopy(message)`. but anyway if you change it so `_onCopy` doesn't take a param you can assign it here directly.